nix/packages.nix

```diff
@@ -96,6 +96,7 @@ in rec {
     scalac   = scala;
     scaladoc = scala;
     scalap   = scala;
+    sbt      = pkgs.sbt;
```

Comment:
Can we please get a comment on why this is required? Presumably for the scala quickstart example?

Reply:
I don't know, I'm off today, this was a request from @leo-da on Slack:

> I want to run a 2 sbt builds that should verify that all published artifacts work with each other

which seemed sensible to me. I suspect once this is in his next step will be to modify the build process to use it. Would that address your issue?

Reply:
@cocreature yes, this is for the `quickstart-scala` so we can validation that the released artifacts all right and the example works!